epoxy-adapter/src/main/java/com/airbnb/epoxy/EpoxyModel.java

```diff
@@ -469,6 +471,23 @@ public int getSpanSize(int totalSpanCount, int position, int itemCount) {
     return 1;
   }
 
+  public EpoxyModel<T> spanSizeCallback(@Nullable SpanSizeCallback spanSizeCallback) {
```

Comment:
I added this so that span size can be determined in a callback instead of inside the model. Otherwise the models generated from views can't handle programmatically changing span size well.